backends/lean/Aeneas/FCongr/FCongr.lean

```diff
@@ -0,0 +1,18 @@
+import Lean
+
+open Lean Meta Parser Elab Tactic
+
+namespace FCongr
+
+syntax (name := fcongr) "fcongr" (ppSpace num)? : tactic
+
+@[tactic fcongr]
+def evalFCongr : Tactic
+| `(tactic| fcongr $[$n?]?) => withMainContext do
+  let hugeDepth := 1000000
+  let depth := n?.map (·.getNat) |>.getD hugeDepth
+  withTransparency .reducible do
```

Comment:
I guess changing the transparency is the trick which makes it work? What is the default reducibility used by `congr`?